xbmc/music/windows/GUIWindowMusicNav.cpp

```diff
@@ -516,9 +516,9 @@ void CGUIWindowMusicNav::GetContextButtons(int itemNumber, CContextButtons &butt
         item->m_bIsFolder && !item->IsVideoDb())
       {
         ADDON::ScraperPtr info;
-        m_musicdatabase.GetScraperForPath(item->GetPath(), info, ADDON::ADDON_SCRAPER_ARTISTS);
-        if (info && info->Supports(CONTENT_ARTISTS))
-          buttons.Add(CONTEXT_BUTTON_INFO_ALL, 21884);
+        if(m_musicdatabase.GetScraperForPath(item->GetPath(), info, ADDON::ADDON_SCRAPER_ARTISTS))
+          if (info && info->Supports(CONTENT_ARTISTS))
```

Comment:
With one if no braces are fine, nested ifs are harder to read and easier to screw up without braces. We have no written rule about it but it's a preference. Same goes for single if where the statement is long and requires being broken up into several lines for readability